examples/all-components/src/ds/mod.rs

```diff
@@ -0,0 +1,38 @@
+mod components;
+mod elements;
+mod theme;
+
+pub use components::*;
+pub use elements::*;
+pub use theme::*;
+
+use freya::prelude::*;
+
+#[component]
+pub fn DsGui(children: Element) -> Element {
+    let theme_signal = use_init_default_theme();
+
+    rsx!(
+        rect{
+            main_align: "center",
+            cross_align: "center",
+            height: "100%",
+            width: "100%",
+            rect {
+                height: "80%",
+                width: "100%",
+                ThemeProvider {
+                    theme: theme_signal()
+                    {children}
+                }
```

Comment:
`ThemeProvider` already calls `use_init_default_theme` internally, we don't need it here.
```suggestion
                {children}
```